docs/wsrep-system-index.md

```diff
@@ -666,20 +666,22 @@ when the node tries to join again but the change will be undone.
 | Dynamic:       | Yes                 |
 | Default Value: | ``OFF`` |
 
-This variable has been announced as deprecated in `5.7.24-31.33`. Defines whether the node should use transparent handling
-of preordered replication events (like replication from traditional source).
-By default, this is disabled.
+This variable has been announced as deprecated in `5.7.24-31.33`. 
+
+Defines whether the node should use transparent handling of preordered replication events (like replication from traditional source). By default, this variable is disabled.
 
-If you enable this variable, such events will be applied locally first
-before being replicated to other nodes in the cluster.
-This could increase the rate at which they can be processed,
-which would be otherwise limited by the latency
-between the nodes in the cluster.
+If you enable this variable, preordered replication events will be applied locally first before being replicated to other nodes in the cluster.
```

Comment:
replication events are first applied locally before the events are replicated to other nodes.

Reply:
updated

Reply:
I don't know where this information came from, but it is not true. Actually, it is the opposite. Events are replicated to the cluster and then applied independently on every node.
The case described above (apply before replication) is for regular replication.

Reply:
I agree with Kamil. We should document the exact way of how it works.